src/Viewer/components/Monaco/MonacoInstance.js

```diff
@@ -100,21 +95,17 @@ function MonacoInstance ({
 }) {
     const {theme} = useContext(ThemeContext);
     const editorRef = useRef(null);
-    const monacoRef = useRef(null);
+    const editorContainerRef = useRef(null);
     const timeoutRef = useRef(null);
-    const [monacoTheme, setMonacoTheme] = useState("customLogLanguageLight");
-
-    const [language, setLanguage] = useState("");
-
-    loader.config({monaco});
+    const getMonacoThemeName = (theme) => (
```

Comment:
Can we move this after the lifecycle methods (before/onmount/useEffect)? That's more similar to our guidelines in other languages where we put things like constructors before generic methods.

Reply:
How about moving it to the position before the `useEffect` that calls `monaco.editor.setTheme`? I referenced `gotoLine`, which is defined right before the two `useEffect`s that are actually calling `gotoLine`.

Reply:
* `gotoLine` (and a lot of the existing code in this repo) doesn't really follow our guidelines. If you'd like to move `gotoLine` after the lifecycle methods, in this PR, that's also fine.
* I would still say we should move it after the `useEffect` calls. Afaik, this will work fine and it stays consistent with our guidelines, reducing the cognitive load for developers who jump into the project later.